### PR TITLE
Add ActivityEditCommand to User Guide

### LINKED ISSUE #381: Add Activity /edit command to user guide

### PR DESCRIPTION
**Pull request by matheril**
Add ActivityEditCommand to User Guide, screenshot included.

Fixes #381

### CODE REVIEW

**Review by warrencxw — CHANGES_REQUESTED** [review]
Minor modification to reduce repetition of information suggested. Otherwise, looks ok to me.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -298,7 +298,47 @@ Example of usage:
...
+>[`activity /list`](#listing-all-activities-in-a-session-activity-list) command.
+> 
+> Split using a cost list: `activity /edit /sid [SESSION_ID] /aid [ACTIVITY_ID] /n [ACTIVITY_NAME] p [PAYER]
+  /i [NAME1 NAME2...] /cl [COST1 COST2...] [</gst [GST_PERCENTAGE]>] [</sc [SERVICE_CHARGE]>]`<br>
+>* `[SESSION_ID]` refers to the unique identifier of the session.
+   >    * The unique identifier for a session can be retrieved with [`session /list`](#listing-all-sessions-session-list)
+          command.
+>* `[ACTIVITY_ID]` refers to the unique identifier of the activity.
+   >      * The unique identifier for an activity can be retrieved with the
+            [`activity /list`](#listing-all-activities-in-a-session-activity-list) command.
```
[the remark this answers is not in the document]
Comment (matheril) [reply docs/UserGuide.md]: Resolved

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Review by froststein — APPROVED** [review]
Looks good to merge

**Pull request merged**